src/ParkChecklist/ParkChecklist.js

```diff
@@ -299,7 +249,7 @@ const ParkChecklist = ({ apiLink }) => {
       path: `/login`,
       element: (
         <div>
-          <Login loggedIn={loggedIn} setLoggedIn={setLoggedIn}/>
+          <Login loggedIn={loggedIn} setLoggedIn={setLoggedIn} />
```

Comment:
In our component refactor branch, we will want to pull this out to the App level with the rest of the routing